README.md

```diff
@@ -78,7 +78,7 @@ Hound supports multiple drivers like Chrome, Firefox, etc but it does not suppor
 To write concurrent acceptance tests with Wallaby, first add it as a dependency to your `mix.exs`:
 
 ```elixir
-{:wallaby, "~> 0.22"}
+{:wallaby, "~> 0.25", only: :test, runtime: false}
```

Comment:
Why runtime: false? :)

Reply:
As a maintainer of the lib, I should probably know this, but I'm not totally sure 🤔.

We specify adding `{:ok, _} = Application.ensure_all_started(:wallaby)` in the test_helper.exs.

They might both be unnecessary now a days.

Reply:
Yup, if you remove the runtime, then you don't need the ensure all started. :)